src/smartpeak/source/io/InputDataValidation.cpp

```diff
@@ -556,4 +597,77 @@ namespace SmartPeak
     oss << "\n";
     return oss.str();
   }
+
+  bool InputDataValidation::prepareToLoad(const Filenames filenames, const std::string& id)
```

Comment:
Is there a reason Filenames is passed by copy and not reference?

Reply:
to be fixed.